Build fails on FreeBSD/ HardenedBSD

```
    Updating crates.io index
    Updating git repository `https://github.com/MarinPostma/pest.git`
thread 'main' panicked at 'byte index 2 is not a char boundary; it is inside 'ϗ' (bytes 1..3) of ` tp`', src/libcore/str/mod.rs:2219:
```

So making long story short, the only issue here is:

```
error[E0425]: cannot find value `CLOCK_BOOTTIME` in crate `libc`
   --> /User/.cargo/registry/src/github.com-1ecc6299db9ec823/sysinfo-0.12.0/src/linux/system.rs:120:43
    |
120 |     if unsafe { libc::clock_gettime(libc::CLOCK_BOOTTIME, &mut up) } == 0 {
    |                                           ^^^^^^^^^^^^^^ help: a constant with a similar name exists: `CLOCK_REALTIME`
    |
   ::: /User/.cargo/registry/src/github.com-1ecc6299db9ec823/libc-0.2.71/src/unix/bsd/freebsdlike/freebsd/mod.rs:430:1
    |
430 | pub const CLOCK_REALTIME: ::clockid_t = 0;
    | ------------------------------------------ similarly named constant `CLOCK_REALTIME` defined here
```

--- a/meilisearch-http/Cargo.toml
+++ b/meilisearch-http/Cargo.toml
@@ -49,7 +49,7 @@ sha2 = "0.8.1"
 siphasher = "0.3.2"
 slice-group-by = "0.2.6"
 structopt = "0.3.12"
-sysinfo = "0.12.0"
+sysinfo = "0.14.5"
 tokio = { version = "0.2.18", features = ["macros"] }
 ureq = { version = "0.12.0", features = ["tls"], default-features = false }
 walkdir = "2.3.1"

